lib/src/main/java/com/springboot/webpush/common/api/Notification.java

```diff
@@ -13,14 +13,14 @@
  * services.
  */
 public class Notification {
-    private final String              title;
-    private final String              body;
-    private final Map<String, Object> data;
-    private final List<Integer>       vibrate;
-    private final List<Action>        actions;
-    private final String              icon;
-    private final String              image;
-    private final boolean             requireInteraction;
+    private final String                   title;
```

Comment:
maybe we want `tag`, `badge`, `timestamp`, `renotify`, too?

Reply:
Added but not tested!!!